src/sorcha/ephemeris/simulation_driver.py

```diff
@@ -95,8 +96,11 @@ def create_ephemeris(orbits_df, pointings_df, args, configs):
     if args.output_ephemeris_file and args.outpath:
         ephemeris_csv_filename = os.path.join(args.outpath, args.output_ephemeris_file)
 
+    verboselog("Building ASSIST ephemeris object.")
     ephem, gm_sun = create_assist_ephemeris(args)
+    verboselog("Furnishing SPICE kernels.")
     furnish_spiceypy(args)
+    verboselog("Generating simulations.")
```

Comment:
Can we be more specific here ASSIST+REBOUND simulations

Reply:
Done, just waiting for the tests to pass then I'll merge.